api/trace/api.go

```diff
@@ -159,12 +159,12 @@ func WithAttributes(attrs ...core.KeyValue) SpanOption {
 	}
 }
 
-// WithRecordEvents enables recording of the events while the span is active.
-// In the absence of this option, RecordEvent is set to false, disabling any recording of
-// the events.
-func WithRecordEvents() SpanOption {
+// WithRecord specifies that the span should be recorded.
+// Note that the implementation may still override this preference,
+// e.g., if the span is a child in an unsampled trace.
+func WithRecord() SpanOption {
```

Comment:
This was removed from the Span start option in the specs.

Reply:
That feels like it should be its own issue (and PR) then. I've filed #188

Reply:
I think the number is wrong (it points to this PR).

Reply:
Oops. #192